KerbalVR_Mod/KerbalVR/InteractionSystem/KerbalVR_Hand.cs

```diff
@@ -372,6 +412,37 @@ protected void Update()
 
 			}
 		}
+
+		private void SyncCollider()
+		{
+			fingertipTransform.position = indexTargetTransform.TransformPoint(CurrentProfile.fingertipOffset);
+			thumbTransform.position = thumbTargetTransform.position;
+			palmTransform.position = palmTargetTransform.TransformPoint(CurrentProfile.gripOffset);
+			
+			fingertipTransform.rotation = indexTargetTransform.rotation;
+			palmTransform.rotation = handObject.transform.rotation;
+
+			// update position of the pinch transform to the middle between the tip of the index finger and the tip of the thumb
+			pinchTransform.position = Vector3.Lerp(fingertipTransform.position, thumbTransform.position, 0.5f);
+		}
+
+		private void SwitchProfile()
+		{
+			Utils.SetLayer(gameObject, UseIVAProfile ? 20 : 0);
+			palmTransform.gameObject.layer = UseIVAProfile ? 20 : 3;
+
+			IVAObject.SetActive(UseIVAProfile);
+			EVAObject.SetActive(!UseIVAProfile);
+
+			fingertipCollider.collider.radius = CurrentProfile.fingertipColliderSize;
+			palmCollider.collider.radius = CurrentProfile.palmColliderSize;
+			pinchCollider.collider.radius = CurrentProfile.pinchColliderSize;
+
+			indexTargetTransform = CurrentHandObject.transform.Find(CurrentProfile.indexTipTransformPath);
+			thumbTargetTransform = CurrentHandObject.transform.Find(CurrentProfile.thumbTipTransformPath);
```

Comment:
This should probably just be re-attaching the thumb, index, and palm collider gameobjects to the new transform

Reply:
Fixed by [c35e6bc](https://github.com/JonnyOThan/Kerbal-VR/pull/172/commits/c35e6bc15153e8ec6391f1a7d6308ce51420088a)